cmd/eval.go

```diff
@@ -663,6 +665,10 @@ func setupEval(args []string, params evalCommandParams) (*evalContext, error) {
 		regoArgs = append(regoArgs, rego.Strict(params.strict))
 	}
 
+	if params.regoV1 {
+		regoArgs = append(regoArgs, rego.SetRegoVersion(ast.RegoV1))
+	}
+
```

Comment:
For both `eval` and `build`, we need to document how the new flag works either in docs or the help text.

Reply:
For sure 👍 . I'm thinking we should update the OPA 1.0 page. 
It's not included here as I didn't want to spend time on writing that text before we locked down the flag name.